src/quick-setup.md

```diff
@@ -30,19 +30,19 @@ Bear in mind, if you plan on using the server purely for [standalone usage via O
 
    ![The first page of the SlimeVR Wizard](assets/img/quick_intro.png)
 
-1. Input your 2.4ghz WiFi credentials, so that your tracker can connect to WiFi, then click **Submit**.
+1. Input your 2.4GHz Wi-Fi credentials, so that your tracker can connect to WiFi, then click **Submit**.
 
    ![Inputting WiFi credentials](assets/img/quick_wifi.png)
 
-1. Plug your trackers in one at a time and turn them on, you should see the process bar on the left update to show the wifi details being stored. Make sure you use the cable included with your trackers, other cables may not be created to send data.
+1. Plug your trackers in one at a time and turn them on, you should see the progress bar on the left update to show the Wi-Fi details being sent. Make sure you use the cable included with your trackers, as other cables may not be suited to send data.
 
 	![Animation of the connection process](assets/img/quick_connectTracker.gif)
 
-1. Once you have connected all your trackers you should see them listed with numbers on the right. If you have forgotten which trackers have yet to plugged in, shaking a powered on tracker will highlight it in the list if it has already been connected. Click **I connected all my trackers** once you are finished.
+1. Once you have connected all your trackers, you should see them listed with numbers on the right. If you have forgotten which trackers have yet to be plugged in, shaking a powered on tracker will highlight it in the list, if it has already been connected. Click **I connected all my trackers** once you are finished.
```

Comment:
```suggestion
1. Once you have connected all your trackers, you should see them listed with numbers on the right. If you have forgotten which trackers have yet to be plugged in, shaking a powered on tracker will highlight it in the list if it has already been connected. Click **I connected all my trackers** once you are finished.
```

Reply:
Yeah I couldn't find a way nice to word it when I was working on this, this is more clear though